### ISSUE TITLE
Cannot access HAI omni units flag values

### ISSUE THREAD

**Issue by davetres** [excaliburpartners/HAILogger#49]
I cannot access the unit flag value from HAI omni II. The units from HAI are defined as light entities in HA and I only see the following attributes:

supported_color_modes: brightness
color_mode: brightness
brightness: 0
friendly_name: TempValue
supported_features: 0

**Comment by rwagoner** [comment]
HAI units 385 and greater are currently implemented as Home Assistant switches (off/on) in OmniLink Bridge.

**Comment by davetres** [comment]
ok. Unfortunately I have a HAI omni IIe and units stop at 128. Is there another way ? Because I can see the value I need in the omnilink bridge logs:

2022-11-02 10:20:12 [OmniLinkBridge.Modules.LoggerModule VRB] UnitStatus 74 HVACAirDiff, Status: 1, Value: 1
2022-11-02 10:20:12 [OmniLinkBridge.Modules.LoggerModule VRB] UnitStatus 79 HotWaterTemp, Status: 49, Value: 49
2022-11-02 10:20:12 [OmniLinkBridge.Modules.LoggerModule VRB] UnitStatus 85 HVACTemp, Status: 225, Value: 225

**Comment by rwagoner** [comment]
I have this resolved for version 1.1.14. You will want to override the flags to the number type in OmniLinkBridge.ini for them to show up in Home Assistant. If you are using MQTT directly the values will be under topic `omnilink/unitX/flag_state`.

```
mqtt_discovery_override_unit = id=74;type=number
mqtt_discovery_override_unit = id=79;type=number
mqtt_discovery_override_unit = id=85;type=number
```

**Comment by davetres** [comment]
Works perfectly! Thank you very much!
